sdcm/tester.py

```diff
@@ -256,6 +262,7 @@ def setUp(self):
         cluster.Setup.reuse_cluster(False)
         if self.monitors.nodes:
             self.prometheusDB = PrometheusDBStats(host=self.monitors.nodes[0].public_ip_address)
+        self.start_time = time.time()
```

Comment:
why do you init `self.start_time` twice?

Reply:
it set twice for better count. First time it initiated at moment, when sct is starting. if setup will failed, report will have the time of start and there will be no exception of missing attribute. 
second time it is initiated when setup is successfully finished. At this moment we have more accurate time, when actually test was started